Makefile

```diff
@@ -529,9 +529,7 @@ test: bins ## Build and run all tests. This target is for local development. The
 	$Q rm -f test.log
 	$Q echo Running special test cases without race detector:
 	$Q go test -v ./cmd/server/cadence/
-	$Q for dir in $(PKG_TEST_DIRS); do \
-		go test $(TEST_ARG) -coverprofile=$@ "$$dir" $(TEST_TAG) | tee -a test.log; \
-	done;
+	$Q go test $(TEST_ARG) -coverprofile=$@ ./... $(TEST_TAG) | tee -a test.log
```

Comment:
`-coverprofile=` needs to be changed, `test` isn't correct here

Reply:
I'm not sure I understand - it's almost the same command as before, just for all test instead of single tests?

Reply:
Also i made sure the generated cover profile is the same as before

Reply:
aah, sorry, I was misreading.  yea this was just kinda weird wasn't it - we should probably change that to `$(UNIT_COVER_FILE)` or something more coverage-looking than a file named `test`...

anyway, unchanged is also good, so 👍